src/api/tick-stream.js

```diff
@@ -1,7 +1,7 @@
 const DerivAPIBasic = require('@deriv/deriv-api/dist/DerivAPIBasic');
 
 const api = new DerivAPIBasic({ endpoint: 'frontend.binaryws.com', lang: 'EN', app_id: 31635 });
-const token = localStorage.getItem('token');
+const token = process.env.REACT_APP_API_KEY;
```

Comment:
Can we give it a different name to the environment variable to specify that it's a deriv api token? Something like `DERIV_API_KEY` would do I guess